lib/build.gradle.kts

```diff
@@ -0,0 +1,183 @@
+import java.net.URI
+
+/*
+ * This file was generated by the Gradle 'init' task.
+ *
+ * This generated file contains a sample Java library project to get you started.
+ * For more details on building Java & JVM projects, please refer to
+ * https://docs.gradle.org/8.4/userguide/building_java_projects.html in the Gradle documentation.
+ */
+
+plugins {
+    // Apply the java-library plugin for API and implementation separation.
+    `java-library`
+    `maven-publish`
+    signing
+}
+
+java {
+    // Apply a specific Java toolchain to ease working on different environments.
+    toolchain {
+        languageVersion.set(JavaLanguageVersion.of(17))
+    }
+    // Generate javadoc and sources jar for maven central publishing
+    withJavadocJar()
+    withSourcesJar()
+}
+
+group = "fr.insee.ddi"
+version = "0.1.0-SNAPSHOT"
+
+repositories {
+    // Use Maven Central for resolving dependencies.
+    mavenCentral()
+    //
+    mavenLocal()
```

Comment:
Utiliser mavenLocal() en premier : évite des appels réseaux inutiles ?

Reply:
Oooh bien vu ! en effet si je comprends bien la doc l'ordre compte : 

https://docs.gradle.org/current/userguide/declaring_repositories.html

> The order of declaration determines how Gradle will check for dependencies at runtime. If Gradle finds a module descriptor in a particular repository, it will attempt to download all of the artifacts for that module from the same repository. You can learn more about the inner workings of [dependency downloads](https://docs.gradle.org/current/userguide/dependency_resolution.html#sec:how-gradle-downloads-deps).

Reply:
8f73dd849ade38a667e5751a9091a166876a970b